apps/web/src/components/NftPreview/NftPreview.tsx

```diff
@@ -108,10 +108,9 @@ function NftPreview({
   const isIFrameLiveDisplay = Boolean(shouldLiveRender && token.media?.__typename === 'HtmlMedia');
```

Comment:
didn't want to rock the boat too much here as this is a critical component for galleries